### DISCUSSION TITLE
Wrong Url on All my Github Pages

### DISCUSSION THREAD

**Discussion by Omobayode6** [community/community#37652] [Other Feature Feedback, Questions, & Ideas]
I use the GitHub student pack to get a free domain name from Namecheap to use for a project and the domain name is automatically mapped to all my GitHub pages causing them to have an error 404, which I don't want.

I want to change them back to the default omobayode6.github.io/repo-name instead of the current omobayode.me/repo-name

I have checked all documentation on the custom domain but can find the solution to mine. In fact the domain name omobayode.me is not showing as a custom domain because I never added it, so I cannot remove it.

Thank you.

![1](https://user-images.githubusercontent.com/99341548/198686764-5c6c0328-c89b-463b-9331-505c15c39e89.PNG)
![2](https://user-images.githubusercontent.com/99341548/198686775-e5ba426e-5318-4598-8c0d-5be50e83f235.PNG)

**Comment by DenverCoder1** [comment]
> I want to change them back to the default omobayode6.github.io/repo-name instead of the current omobayode.me/repo-name


There is a file committed to the repository called `CNAME` which github uses to decide your custom domain. If you delete that, it should go back to the default.

**Reply by Omobayode6** [reply]
I read about that too but the funniest thing is that all the repository has no CNAME file created in the root folder.

![image](https://user-images.githubusercontent.com/99341548/198833775-4b4fa0e4-6503-4cf6-8f4c-ba4b02fdfb94.png)

**Comment by airtower-luna** [comment]
Your `omobayode6.github.io` repository creates a *user site*, see [Types of GitHub Pages sites](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites). If you create a Pages site without a custom domain it'll always be under the same domain as your user site.

Renaming the `omobayode6.github.io` repository should work: It'll still have its custom domain but won't be the user site any more, so other sites you have should use the `github.io` domain again.

**Reply by Omobayode6** [reply]
This is the solution to the problem. This works!!!! 

Thank you so much @airtower-luna, you cannot imagine the amount of documentation I have read while trying to solve this problem. I have been looking for CNAME everywhere not knowing that an Omobayode.github.io repository has been created.

![image](https://user-images.githubusercontent.com/99341548/198834489-9d24b069-5a8f-410e-a372-f3975e19c014.png)
